charts/k8s-infra/values.yaml

```diff
@@ -90,9 +90,22 @@ presets:
     samplingThereafter: 500
   otlpExporter:
     enabled: true
+  ownTelemetry:
+    # OTLP HTTP endpoint to send own telemetry data to.
+    endpoint: ""
+    # Whether to use insecure mode.
+    insecure: true
+    # Whether to skip verifying the certificate.
+    insecureSkipVerify: true
+    traces:
+      enabled: false
+    metrics:
+      enabled: false
+    logs:
+      enabled: false
   logsCollection:
     enabled: true
-    startAt: beginning
+    startAt: end
```

Comment:
@prashant-shahi  this should be end otherwise if the collector restarts then duplicate data will be ingested

Reply:
yes, that's right. Collector restart will cause the duplicate logs data collection.

Nice catch, Nitya!